Library/Homebrew/extend/os/ENV/std.rb

```diff
@@ -0,0 +1,5 @@
+require "ENV/std"
```

Comment:
Looks like there's a `require "extend/os/ENV/std"` missing that would need to be added to the end of the `extend/ENV/std.rb` file.